lxdock/conf/schema.py

```diff
@@ -12,6 +12,7 @@
     'lxc_config': {Extra: str},
     'mode': In(['local', 'pull', ]),
     'privileged': bool,
+    'profiles': [str, ],
```

Comment:
@ellmetha  Here's more information on profile name requirements:

General: profile names cannot contain slashes, and cannot be `.` or `..`

Code: https://github.com/lxc/lxd/blob/74e96210ad59b722a602a26579a8b8b561e8251f/lxd/profiles.go#L72-L78
Discussion: https://lists.linuxcontainers.org/pipermail/lxc-devel/2016-August/014576.html

Tests:
```console
$ lxc profile create "a/fa/b/a 2qijgq290 q2398gei@$2132"
error: Profile names may not contain slashes
$ lxc profile create "2qij.gq290 q2398gei@$2132"
Profile 2qij.gq290 q2398gei@132 created
$ lxc profile create "2qij..gq290 q2398gei@$2132"
Profile 2qij..gq290 q2398gei@132 created
$ lxc profile create ".."
error: Invalid profile name '..'
$ lxc profile create "."
error: Invalid profile name '.'
$ lxc profile create "..."
Profile ... created
```

However, I didn't find this in any formal documentation. Perhaps we still leave it as `str`, and let lxd throw the error?